core/services/workflows/engine.go

```diff
@@ -342,6 +346,7 @@ func (e *Engine) init(ctx context.Context) {
 	}
 
 	e.logger.Info("engine initialized")
+	logCustMsg(e.cma, "workflow registered", e.logger)
```

Comment:
Are you missing a message when we fail to initialize, eg. on line 327?

Reply:
Do we want a message on failure on each possible `err` occurence?